client/app/account/contacts/service/user-contacts-service.routes.js

```diff
@@ -1,14 +1,57 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('app.account.contacts.services', {
+  const stateName = 'app.account.contacts.services';
+
+  $stateProvider.state(stateName, {
     url: '/services?serviceName&category',
     component: 'accountContactsService',
     translations: {
       format: 'json',
       value: ['.', '..'],
     },
+    params: {
+      serviceName: {
+        value: null,
+        dynamic: true,
+      },
+      category: {
+        value: null,
+        dynamic: true,
+      },
+    },
     resolve: {
+      criteria: ($transition$, $translate) => {
+        const criteria = [];
+
+        return $translate.refresh().then(() => {
+          if ($transition$.params().serviceName) {
```

Comment:
Noticed that you're targeting `master` but you can use destructuring assignment.

```js
const { category, serviceName } = $transition$.params();
```